### PR TITLE
STY: fstrings in io.pytables

### PR DESCRIPTION
**Pull request by jbrockmendel**
honestly not sure why ive got such a bee in my bonnet about this file

### CODE REVIEW

File: pandas/io/pytables.py [hunk 1]
```diff
@@ -1931,8 +1910,7 @@ def validate_attr(self, append: bool):
             existing_kind = getattr(self.attrs, self.kind_attr, None)
             if existing_kind is not None and existing_kind != self.kind:
                 raise TypeError(
-                    "incompatible kind in col [{existing} - "
-                    "{self_kind}]".format(existing=existing_kind, self_kind=self.kind)
+                    f"incompatible kind in col [{existing_kind} - " f"{self.kind}]"
```
Comment (simonjayhawkins) [inline pandas/io/pytables.py]: concatenate

File: pandas/io/pytables.py [hunk 2]
```diff
@@ -4012,8 +3961,7 @@ def process_filter(field, filt):
                             return obj.loc(axis=axis_number)[takers]
 
                     raise ValueError(
-                        "cannot find the field [{field}] for "
-                        "filtering!".format(field=field)
+                        f"cannot find the field [{field}] for " "filtering!"
```
Comment (simonjayhawkins) [inline pandas/io/pytables.py]: concatenate

**Review by simonjayhawkins — APPROVED** [review]
@jbrockmendel minor nits otherwise lgtm

**Comment by jreback** [comment]
can you rebase

**Comment by jbrockmendel** [comment]
rebased+green

**Comment by WillAyd** [comment]
Thanks @jbrockmendel

**Pull request merged**